identity-account/src/storage/stronghold.rs

```diff
@@ -83,7 +84,8 @@ impl Stronghold {
   }
 }
 
-#[async_trait::async_trait]
+#[cfg_attr(feature = "wasm", async_trait(?Send))]
+#[cfg_attr(not(feature = "wasm"), async_trait)]
```

Comment:
We won't be exposing `Stronghold` to Wasm anyway, so we can actually remove the `cfg_attr`s completely here and go with `async_trait` directly.